keyboards/dztech/dz68rgb/rules.mk

```
@@ -0,0 +1,10 @@
+MCU  = STM32F303
+BACKLIGHT_ENABLE = no
+BOOTMAGIC_ENABLE = lite           # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes              # Mouse keys
+EXTRAKEY_ENABLE = yes              # Audio control and System control
+CONSOLE_ENABLE = no                # Console for debug
+COMMAND_ENABLE = no                # Commands for debug and configuration
+NKRO_ENABLE = yes                 # USB Nkey Rollover
+AUDIO_ENABLE = no
+RGB_MATRIX_ENABLE = yes     # Use RGB matrix
```

By adding this line, you can type in

`make dztech/dz68rgb:yourcommunitylayout` and it will build your community layout if you have one. 

```suggestion
RGB_MATRIX_ENABLE = yes     # Use RGB matrix

LAYOUTS = 68_ansi
```